Cargo.toml

```diff
@@ -25,8 +24,7 @@ systemstat = { version = "0", optional = true }
 xcb = { version = "0.8", features = ["xkb"], optional = true }
 serde = "0.8"
 serde_json = "0.8"
-alsa = { version = "0.1.10", optional = true }
-libc = { version = "0.2.29", optional = true }
 
-[patch.crates-io]
-systemstat = { git = "https://github.com/myfreeweb/systemstat" }
+[target.'cfg(target_os = "linux")'.dependencies]
```

Comment:
Removed the `unix_alsa` feature, this seems simpler.

Reply:
Yeah that's fine for now, though I'd imagine someone might prefer e.g. a pulseaudio-only build in the future.